examples/examples_test.go

```diff
@@ -3,126 +3,215 @@
 package examples
 
 import (
-	"io/ioutil"
-	"net/http"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/lambda"
+	"github.com/stretchr/testify/assert"
 	"os"
 	"path"
 	"path/filepath"
 	"testing"
-	"time"
 
-	"github.com/stretchr/testify/assert"
-
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/session"
-	"github.com/aws/aws-sdk-go/service/lambda"
 	"github.com/pulumi/pulumi/pkg/testing/integration"
 )
 
-func TestExamples(t *testing.T) {
-	envRegion := os.Getenv("AWS_REGION")
-	if envRegion == "" {
-		t.Skipf("Skipping test due to missing AWS_REGION environment variable")
-	}
+func TestAccMinimal(t *testing.T) {
+	test := getJSBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir: path.Join(getCwd(t), "minimal"),
+		})
 
-	cwd, err := os.Getwd()
-	if !assert.NoError(t, err, "expected a valid working directory: %v", err) {
-		return
-	}
+	t.Run(test.Dir, func(t *testing.T) {
+		integration.ProgramTest(t, &test)
+	})
```

Comment:
We don’t actually need this `t.Run` anymore - right?  That causes this to be a child test, but we don’t need that as there is just one test?

Reply:
Ah you are correct - sorry, my misunderstanding! Just pushed an update to this to remove those t.Runs